BE/OwnershipAndControl/Executives.rdf

```diff
@@ -256,171 +307,93 @@
 		<fibo-fnd-utl-av:synonym>corporate law</fibo-fnd-utl-av:synonym>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-be-oac-exec;CompanySecretary">
 		<rdfs:subClassOf rdf:resource="&fibo-be-oac-exec;CorporateOfficer"/>
 		<rdfs:label>company secretary</rdfs:label>
-		<skos:definition>Officer appointed by the directors of a firm as responsible for ensuring that firm&apos;s legal obligations under the corporate legislation are complied with. His or her formal duties include (1) calling meetings, (2) recording minutes of the meetings, (3) keeping statutory record books, (4) proper payment of dividend and interest payments, and (5) proper drafting and execution of agreements, contracts, and resolutions. A company secretary is not automatically an employee of the firm and, if employed with executive responsibilities, not be its director shareholder. If a firm has only two directors, one may act as its secretary; but a sole director may not. A firm (such as of accountants) may not act as a company secretary for any firm. Called corporate secretary in the US.</skos:definition>
-		<skos:editorialNote>Functionary but also a signer of documentation in the company. Does not have powes to do anything without instruction from the board. THe rol arsies from legal obligations iposed by the company. responsible to legislative authorities fo rthe company meeting those legal obligations. Role is that they gain a degree of control but only in the fact that they must report according to rules established by theirlegislative jurisdiction. Example: Delaware: you need a designated corporate secretary who is responsible for submitting certain documents etc.</skos:editorialNote>
-		<fibo-fnd-utl-av:definitionOrigin rdf:datatype="&xsd;anyURI">http://www.businessdictionary.com/definition/company-secretary.html</fibo-fnd-utl-av:definitionOrigin>
+		<skos:definition>corporate officer appointed by the directors of an organization, responsible for ensuring compliance with legal obligations related to corporate governance</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">http://www.businessdictionary.com/definition/company-secretary.html</fibo-fnd-utl-av:adaptedFrom>
+		<fibo-fnd-utl-av:explanatoryNote>His or her formal duties include (1) calling meetings, (2) recording minutes of the meetings, (3) keeping statutory record books, (4) proper payment of dividend and interest payments, and (5) proper drafting and execution of agreements, contracts, and resolutions. A company secretary is not automatically an employee of the firm and, if employed with executive responsibilities, not be its director shareholder. If a firm has only two directors, one may act as its secretary; but a sole director may not. A firm (such as of accountants) may not act as a company secretary for any firm. Called corporate secretary in the US.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
what's "director shareholder"?

Reply:
I think there was a slash between the two that the serializer may have removed.  But the distinction isn't one I am familiar with - a corporate secretary is usually an employee rather than a director, but I don't know if there is a legal prohibition from being both.

Reply:
I removed most of that part of the note - our corporate secretary was our attorney for Sandpiper - I think the same is true for OMG, in fact.  The note would have ruled that out.  Maybe that's a jurisdiction specific thing?